### ISSUE TITLE
General Lightbeam information

### ISSUE THREAD

**Issue by grinapo** [mozilla/lightbeam#614]
I went and checked Lightbeam information webpage to see
- how to use it, and
- what happens to the data I enable to send and
- how to see the results of the crowdsourced data.

There's the main page with basically an advertisement leaflet about how fantastic L. is, and how useful it is. Okay. It leads to an about page which explains history and contributors background, okay. That's all.

There seem to be no info on usage (which is a problem since it contains a few functions only by clicking to various places or selecting nondescript icons with meaningless hovers).

There seem to be no indication about exactly where the data goes and who does what with it.

There seem to be no indication on how could I, as the leaflet explained with soothing pictures, browse the crowdsourced data to make my live better.

Since you kind of target the privacy aware niche of the population it's kind of weird that there's such amount of silence about the data collection, usage and possible retrieval. (But yeah thanks for the data format, still some exact examples/samples could be justified there either.)

And, well, I see lots of open issues around with zero comments. This is possibly not a very good sign, kind of suggest no support activity and this seems to be a Mozilla sponsored project and not a one-man one.

**Comment by bendover22** [comment]
No, lots of open issues w/ no replies (from devs or others with detailed knowledge) isn't a good sign.  Like possibly there's no backing from Mozilla for development.
Maybe the media ran w/ the hype about Lightbeam themselves?  It has good potential, but as I see it currently (v1.1.0, in Fx 34 Win), it's embarrassing to Mozilla for (anyone) to be hyping an addon that has so many flaws.  Some, extremely glaring.

I haven't found basic documentation on things like how to interpret the data shown.
Being in a class of its own, the addon really needs instructions.  
If anyone knows of such documentation, please post it here & perhaps in Mozillazine forums (Extensions), on the Lightbeam mailing list lightbeam@mozilla.org, on the MAO addons site, etc.

**Comment by fmarier** [comment]
If anybody wants to help produce better documentation, we'd love to have it.

Note that the reporting functionality has been removed from Lightbeam. We're no longer collecting Lightbeam data on our servers.

**Comment by grinapo** [comment]
The problem with asking for documentation is that you have plenty of people actually _looking_ for it since they do not know what they wanted to know. :-) I cannot write documentation for unknown things. I could write, for example, questions which should be a answered and this could form a FAQ but this expects someone to be able to answer.

**Comment by fmarier** [comment]
Having a list of questions would be a great start. The next step will be to figure out how to expose it (along with the answers) to users.

**Comment by tukoz** [comment]
GH Wiki would be a great starting place, wouldn't it?
